draft-ietf-ipsecme-ikev2-multiple-ke-00.xml

```diff
@@ -870,15 +886,34 @@ KEYMAT = prf+ (SK_d, SK(0) | Ni | Nr | SK(1) |  ... SK(n))
             messages are used to carry the key exchange payload.  If classical key exchange method is used in
             the IKE_SA_INIT message, the very first Child SA created in IKE_AUTH will offer no resistance against the
             quantum threats.  Consequently, if the peers' local policy requires that all Child SAs to be post-quantum
-            secure, then the peers can avoid creating the very first Child SA by adopting <xref target="RFC6023"/>.  In this
-            case, the peers exchange CHILDLESS_IKEV2_SUPPORTED notification in the IKE_SA_INIT exchange and the desired
-            Child SA can be created with CREATE_CHILD_SA using additional key exchanges.
-            </t>
+            secure, then the peers can avoid creating the very first Child SA by adopting <xref target="RFC6023"/>.  In this case, the initiator sends two types of
+            proposal in the IKE_SA_INIT request, one with and another one without additional
+            key exchanges.  The responder chooses the latter proposal type and includes
+            CHILDLESS_IKEV2_SUPPORTED notification in the IKE_SA_INIT response.  Assuming that
+            the initiator supports childless IKE SA extension, then both peers performs
+            the modified IKE_AUTH exchange described in <xref target="RFC6023"/> and no
+            child SA is created in this exchange.  The peers should then immediately rekey
+            the IKE SA and subsequently create the child SAs, all with additional key
+            exchanges using CREATE_CHILD_SA exchange.</t>
+
+            <t>It is also possible for the initiator to send proposals without additional key
+            exchange in the IKE_SA_INIT message and in this instance, the responder will have
```

Comment:
Again, I'd clarify that these are "Additional Key Exchange  transform(s)", not IKE exchanges :-)